data/src/main/java/io/gnosis/kouban/data/repositories/SafeRepository.kt

```diff
@@ -192,7 +192,7 @@ class SafeRepository(
                 val deploymentArgs = GnosisSafe.Setup.decodeArguments(deploymentArgsEncoded)
                 SafeInfoDeployment(
                     deploymentMastercopy,
-                    deploymentArgs.fallbackhandler,
```

Comment:
Fallback handlers are only defined for 1.1.1?

Reply:
yes